charts/incubator/firezone/questions.yaml

```diff
@@ -331,8 +331,6 @@ questions:
                   type: int
                   default: 568
 # Include{resources}
-# Include{metrics}
-# Include{prometheusRule}
```

Comment:
I think its better to leave me there but change templates to be disabled by default

Reply:
The app doesnt have any metrics but I'll look at traefik to disable then